lib/matplotlib/tests/test_mlab.py

```diff
@@ -1388,32 +1296,37 @@ def createStim(self, fstims, iscomplex, sides, nsides, len_x=None,
         if iscomplex:
             y = y.astype('complex')
 
-        self.Fs = Fs
-        self.sides = sides
-        self.fstims = fstims
+        # Interestingly, the instance on which this fixture is called is not
+        # the same as the one on which a test is run. So we need to modify the
+        # class itself when using a class-scoped fixture.
```

Comment:
Yea, but that would require changing all the `self.something` in the tests, and this PR is already gigantic...